app/assets/stylesheets/application.scss

```diff
@@ -240,6 +240,25 @@ main {
   background-color: rgb(226, 225, 253);
 }
 
+.levels-fontsize {
```

Comment:
bad class name

Reply:
what do i name it then ? levels-navbar is ok?